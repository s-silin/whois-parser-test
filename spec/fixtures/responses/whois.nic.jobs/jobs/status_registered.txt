   Domain Name: MILITARY2DRIVER.JOBS
   Registry Domain ID: 122283906_DOMAIN_JOBS-VRSN
   Registrar WHOIS Server: whois-all.nameshare.com
   Registrar URL: http://www.nameshare.com
   Updated Date: 2018-04-25T05:45:54Z
   Creation Date: 2016-04-22T20:45:45Z
   Registry Expiry Date: 2019-04-22T20:45:45Z
   Registrar: Name Share, Inc
   Registrar IANA ID: 667
   Registrar Abuse Contact Email:
   Registrar Abuse Contact Phone:
   Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
   Registry Registrant ID: REDACTED FOR PRIVACY
   Registrant Name: REDACTED FOR PRIVACY
   Registrant Organization: Holland New Penn Reddaway
   Registrant Street: REDACTED FOR PRIVACY
   Registrant City: REDACTED FOR PRIVACY
   Registrant State/Province: OH
   Registrant Postal Code: REDACTED FOR PRIVACY
   Registrant Country: US
   Registrant Phone: REDACTED FOR PRIVACY
   Registrant Phone Ext: REDACTED FOR PRIVACY
   Registrant Fax: REDACTED FOR PRIVACY
   Registrant Fax Ext: REDACTED FOR PRIVACY
   Registrant Email: Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain.
   Registry Admin ID: REDACTED FOR PRIVACY
   Admin Name: REDACTED FOR PRIVACY
   Admin Organization: REDACTED FOR PRIVACY
   Admin Street: REDACTED FOR PRIVACY
   Admin City: REDACTED FOR PRIVACY
   Admin State/Province: REDACTED FOR PRIVACY
   Admin Postal Code: REDACTED FOR PRIVACY
   Admin Country: REDACTED FOR PRIVACY
   Admin Phone: REDACTED FOR PRIVACY
   Admin Phone Ext: REDACTED FOR PRIVACY
   Admin Fax: REDACTED FOR PRIVACY
   Admin Fax Ext: REDACTED FOR PRIVACY
   Admin Email: Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain.
   Registry Tech ID: REDACTED FOR PRIVACY
   Tech Name: REDACTED FOR PRIVACY
   Tech Organization: REDACTED FOR PRIVACY
   Tech Street: REDACTED FOR PRIVACY
   Tech City: REDACTED FOR PRIVACY
   Tech State/Province: REDACTED FOR PRIVACY
   Tech Postal Code: REDACTED FOR PRIVACY
   Tech Country: REDACTED FOR PRIVACY
   Tech Phone: REDACTED FOR PRIVACY
   Tech Phone Ext: REDACTED FOR PRIVACY
   Tech Fax: REDACTED FOR PRIVACY
   Tech Fax Ext: REDACTED FOR PRIVACY
   Tech Email: Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain.
   Name Server: NS-1186.AWSDNS-20.ORG
   Name Server: NS-1555.AWSDNS-02.CO.UK
   Name Server: NS-342.AWSDNS-42.COM
   Name Server: NS-614.AWSDNS-12.NET
   DNSSEC: unsigned
   Registry Billing ID: REDACTED FOR PRIVACY
   Billing Name: REDACTED FOR PRIVACY
   Billing Organization: REDACTED FOR PRIVACY
   Billing Street: REDACTED FOR PRIVACY
   Billing City: REDACTED FOR PRIVACY
   Billing State/Province: REDACTED FOR PRIVACY
   Billing Postal Code: REDACTED FOR PRIVACY
   Billing Country: REDACTED FOR PRIVACY
   Billing Phone: REDACTED FOR PRIVACY
   Billing Phone Ext: REDACTED FOR PRIVACY
   Billing Fax: REDACTED FOR PRIVACY
   Billing Fax Ext: REDACTED FOR PRIVACY
   Billing Email: Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain.
   URL of the ICANN Whois Inaccuracy Complaint Form: https://www.icann.org/wicf/
>>> Last update of WHOIS database: 2018-07-25T16:42:39Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

NOTICE: The expiration date displayed in this record is the date the
registrar's sponsorship of the domain name registration in the registry is
currently set to expire. This date does not necessarily reflect the
expiration date of the domain name registrant's agreement with the
sponsoring registrar.  Users may consult the sponsoring registrar's
Whois database to view the registrar's reported date of expiration
for this registration.

TERMS OF USE: You are not authorized to access or query our Whois
database through the use of electronic processes that are high-volume and
automated except as reasonably necessary to register domain names or
modify existing registrations; the Data in VeriSign's ("VeriSign") Whois
database is provided by VeriSign for information purposes only, and to
assist persons in obtaining information about or related to a domain name
registration record. VeriSign does not guarantee its accuracy.
By submitting a Whois query, you agree to abide by the following terms of
use: You agree that you may use this Data only for lawful purposes and that
under no circumstances will you use this Data to: (1) allow, enable, or
otherwise support the transmission of mass unsolicited, commercial
advertising or solicitations via e-mail, telephone, or facsimile; or
(2) enable high volume, automated, electronic processes that apply to
VeriSign (or its computer systems). The compilation, repackaging,
dissemination or other use of this Data is expressly prohibited without
the prior written consent of VeriSign. You agree not to use electronic
processes that are automated and high-volume to access or query the
Whois database except as reasonably necessary to register domain names
or modify existing registrations. VeriSign reserves the right to restrict
your access to the Whois database in its sole discretion to ensure
operational stability.  VeriSign may restrict or terminate your access to the
Whois database for failure to abide by these terms of use. VeriSign
reserves the right to modify these terms at any time.
