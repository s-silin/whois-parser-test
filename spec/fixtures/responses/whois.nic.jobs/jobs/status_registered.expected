#domain
  %s == "military2driver.jobs"

#domain_id
  %s == "122283906_DOMAIN_JOBS-VRSN"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2016-04-22 20:45:45 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2018-04-25 05:45:54 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2019-04-22 20:45:45 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == "667"
  %s.name         == "Name Share, Inc"
  %s.organization == 'Name Share, Inc'
  %s.url          == "http://www.nameshare.com"


#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.google.com"
  %s[0].ipv4 == nil
  %s[0].ipv6 == nil
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.google.com"
  %s[1].ipv4 == nil
  %s[1].ipv6 == nil
%s[1] %CLASS{nameserver}
  %s[1].name == "ns-342.awsdns-42.com"
  %s[1].ipv4 == nil
  %s[1].ipv6 == nil
%s[1] %CLASS{nameserver}
  %s[1].name == "ns-614.awsdns-12.net"
  %s[1].ipv4 == nil
  %s[1].ipv6 == nil

